Automattic/jetpack
# Form: Add style variations

Fixes #28166

## Proposed changes:
* Includes 2 new Form styles: `Outlined` and `Animated`

### Other information:


## Jetpack product discussion
pcsBup-Gl-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a post and add a Form block
* Change the form style in the Sidebar to the new styles
* Add some fields and play with the form to ensure everything works as expected
